### PR TITLE
chore: bump gogo/protobuf

### PR DESCRIPTION
**Pull request by robert-zaremba**
regen-network/protobuf was updated at the beginning of the year (9 months ago). Llet's keep this dependency synced in Osmosis as well.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -17,19 +18,17 @@ require (
 	github.com/spf13/cobra v1.1.3
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.0
-	github.com/tendermint/tendermint v0.34.12
+	github.com/tendermint/tendermint v0.34.13
```
Comment (ValarDragon) [inline go.mod:21]: Can we hold off on this? It was reported to be causing some issues when used on mainnet.

File: go.mod [hunk 2]
```diff
@@ -17,19 +18,17 @@ require (
...
+	google.golang.org/grpc v1.40.0
+	google.golang.org/protobuf v1.27.0
 	gopkg.in/yaml.v2 v2.4.0
 )
 
 replace google.golang.org/grpc => google.golang.org/grpc v1.33.2
 
-replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.2-alpha.regen.4
-
-replace github.com/tendermint/tendermint => github.com/tendermint/tendermint v0.34.12
```
Comment (ValarDragon) [inline go.mod:32]: Why was this replace removed?

Comment (robert-zaremba) [reply go.mod:32]: you don't need to replace tendermint - it's already fixed in the "require" section.

Comment (ValarDragon) [reply go.mod:32]: We should still be doing this just in case dependencies use other versions, no?

File: go.mod (hunk 1 above)
Comment (robert-zaremba) [reply go.mod:21]: I thought tendermint 0.34.12 has problems. `..13` is needed for IAVL update, which has good performance improvements.

Comment (robert-zaremba) [reply go.mod:21]: you are right, Bez confirmed.

File: go.mod (hunk 2 above)
Comment (robert-zaremba) [reply go.mod:32]: Yes - it replaces all versions. It looks like a hack. You are depending on your own version of SDK - maybe it's better to fix the version you want there? Anyway - I will rollback that change and in the future you can consider removing it.

**Review by ValarDragon — APPROVED** [review]
Thank you for this!

**Pull request merged**